quesma/model/bucket_aggregations/date_histogram.go

```diff
@@ -64,6 +65,9 @@ func (query *DateHistogram) TranslateSqlResponseToJson(rows []model.QueryResultR
 		)
 	}
 
+	// TODO: implement default  when query.minDocCount == DefaultMinDocCount
+	// all buckets between the first bucket that matches documents and the last one are returned
```

Comment:
I don't understand this TODO. AFAIK default is just `1` and what you've described is what happens when it's `0`, and it's already implemented.